docs/build.md

```diff
@@ -72,32 +72,19 @@ git clone https://github.com/unicode-org/cldr.git
-
-For people who need `mvn test` or other subsequent Maven targets to succeed, a temporary workaround would be the following (which points the generated files directory to find the test input files from the repository sources):
-
-For out-of-source builds:
-```
-ln -s ../../src/unicodetools/data/security unicodetools/mine/Generated/security
-```
-For in-source builds:
-```
-cd <unicodetools-repo-root>; ln -s unicodetools/data/security output/Generated/security
-```
-
-This step to create a symbolic link on the file system is not necessary to run individual tools in Unicode Tools, nor is it intended to last long-term as we refactor code to establish stronger invariants and tests.
+Currently, some tests run on the generated output files of a tool (ex: in order to test the validity of the output files). After converting these tests into standard JUnit tests, these unit tests are then run in isolation by default. Our code has been updated to support this behavior because it [now checks](https://github.com/unicode-org/unicodetools/commit/aa6d11c57fe8bbd20484d3a36123c8948e363262) for generated files in the `Generated` directory, and falls back to the repository's checked-in version when a command does not invoke the generation of a new version.
 
 #### Java System properties used in Unicode Tools
 
 (Note: The following example values for Java system properties are paths to local working copies that are organized using the out-of-source build workspace layout, as described above.)
 
-| Property                | Example Value                                        |
-|-------------------------|------------------------------------------------------|
-| CLDR_DIR                | /usr/local/google/home/mscherer/cldr/mine/src        |
-| IMAGES_REPO_DIR         | /usr/local/google/home/mscherer/images/mine/src      |
-| UNICODETOOLS_REPO_DIR   | /usr/local/google/home/mscherer/unitools/mine/src    |
-| UNICODETOOLS_OUTPUT_DIR | /usr/local/google/home/mscherer/unitools/mine        |
-| UVERSION                | 14.0.0                                               |
+| Property                | Example Value                                                  |
+|-------------------------|----------------------------------------------------------------|
+| CLDR_DIR                | /usr/local/google/home/mscherer/cldr/mine/src                  |
+| IMAGES_REPO_DIR         | /usr/local/google/home/mscherer/images/mine/src                |
+| UNICODETOOLS_REPO_DIR   | /usr/local/google/home/mscherer/unitools/mine/src              |
+| UNICODETOOLS_GEN_DIR    | /usr/local/google/home/mscherer/unitools/mine/output/Generated |
```

Comment:
```suggestion
| UNICODETOOLS_GEN_DIR    | /usr/local/google/home/mscherer/unitools/mine/Generated |
```
Let's drop the "output" layer in the out-of-source version.

Reply:
Done.  I think my previous attempt at doing this at the end of a previous PR was incomplete / inconsistent somehow and caused the CLI instructions to not actually work. Anyways, the new CI workflow is still passing in the PR.